Puzzlepart/did
# Reintroduced last year report

### Your checklist for this pull request
 
### Review checklist

### Description
Please describe your pull request.

### How to test
1. Check out locally with [gh](https://github.com/cli/cli)
2. go to reports
3. click "last year"
4. observe that it takes 25mb / ~30s
5. you now have last years report

### Related issues
Closes #1029
